nextcord/client.py

```diff
@@ -1643,3 +1661,14 @@ def persistent_views(self) -> Sequence[View]:
         .. versionadded:: 2.0
         """
         return self._connection.persistent_views
+
+    @property
+    def scheduled_events(self) -> List[ScheduledEvent]:
+        """List[ScheduledEvent]: A list of scheduled events
```

Comment:
Why isn't this denoted in the docstring guide as returns

```
Returns
--------
List[ScheduledEvent]
 A list of scheduled events
```

Reply:
properties can be done like this, see persistent_views for example